pandas/_libs/tslibs/offsets.pyx

```diff
@@ -15,7 +15,7 @@ np.import_array()
 
 from util cimport is_string_object
 
-from pandas._libs.tslib import pydt_to_i8
+from pandas._libs.tslib import pydt_to_i8, monthrange
```

Comment:
prob should move ``monthrange`` and everything it calls to offsets (and then you can cimport these to tslib.pyx)

Reply:
Eventually.  We've got a few more of these left to go.

Reply:
k, add to the list